### ISSUE TITLE
hfiber categories

### ISSUE THREAD

**Issue by Alizter** [mikeshulman/HoTT#26]
So the hfiber of a X-coherent Y-functor should probably be itself a X-coherent Y-category. I'm not sure how this relates to notions of Grothendieck fibration,

**Comment by mikeshulman** [comment]
I would expect the coherence to go down by one.  But I don't think this is related to fibrations much.

**Comment by Alizter** [comment]
Yes, the coherence does go down I just noticed. It might be related to Grothendieck fibrations in that for f : A -> B,  hfiber f becomes a functor from B to Type. Which is perhaps contravariant I would need to check. I have seen people talk about Grothendieck fibrations as pseudofunctors C^op -> Cat.

**Comment by mikeshulman** [comment]
The fiber of an arbitrary functor isn't functorial in the codomain; at most it is a displayed category.
